README.md

```diff
@@ -15,6 +16,12 @@ a Geographic Information System used for geospatial data management and
 analysis, image processing, graphics/map production, spatial modeling, and
 visualization.
 
+## Jupyter Notebook
+Launch this repository in binder and experiment with GRASS's Python API in Jupyter Notebooks by clicking the button below:
+
+[![Binder](https://camo.githubusercontent.com/581c077bdbc6ca6899c86d0acc6145ae85e9d80e6f805a1071793dbe48917982/68747470733a2f2f6d7962696e6465722e6f72672f62616467655f6c6f676f2e737667)](https://mybinder.org/v2/gh/OSGeo/grass/HEAD?filepath=lab%2Ftree%2Fmaster%2Fdoc%2Fnotebooks%2Fexample_notebook.ipynb)
```

Comment:
The correct URL is:

```suggestion
[![Binder](https://camo.githubusercontent.com/581c077bdbc6ca6899c86d0acc6145ae85e9d80e6f805a1071793dbe48917982/68747470733a2f2f6d7962696e6465722e6f72672f62616467655f6c6f676f2e737667)](https://mybinder.org/v2/gh/OSGeo/grass/master?urlpath=lab%2Ftree%2Fdoc%2Fnotebooks%2Fexample_notebook.ipynb)
```

...well, that's at least the JupyterLab and file path part. The HEAD versus master part (right after `grass/`) is not 100% clear to me as I don't see it in the documentation, but my understanding is that HEAD in this case is the tip of default branch, hence the same as master. It is, however, on master branch, so saying master is probably more expected.